hyperspy/signal.py

```diff
@@ -244,145 +244,6 @@ def __getitem__(self, name):
 class MVATools(object):
     # TODO: All of the plotting methods here should move to drawing
```

Comment:
While we are at tidying things up, should I moved all plotting functions to a `MVADrawing` class in `drawing.signal`?